app/routes/endemics/declaration.js

```diff
@@ -86,8 +87,10 @@ module.exports = [{
         throw boom.internal()
       }
 
+      console.log(application, application.organisation)
       return h.view('endemics/confirmation', {
         reference: applicationReference,
+        newUser: appliedBefore.NEW_USER === application.organisation.appliedBefore,
```

Comment:
I can see  **newUser** property will contain boolean suggest changing it to **isNewUser** for better code readability